submissions/rubyAdeniregun/Database/Queries.sql

```diff
@@ -0,0 +1,33 @@
+/* Question 1: Query to find the total number of passengers that survived 
+The total number of passengers who survived = 342. */
+SELECT * FROM accident_records WHERE passenger_survived LIKE 1; 
+
+
+/* Question2: Query to find the total number of passengers who did not survive
+The total number of passengers who did not survive = 549. */
+SELECT * FROM accident_records WHERE passenger_survived LIKE 0; 
+
+
+/* Question 3: Query to get the name and sex of passengers under 
+the age of 27 that embarked at Queenstown and Cherbourg? */
+ SELECT registered_passengers.full_name, registered_passengers.sex 
+ FROM registered_passengers LEFT JOIN passengers_records 
+ ON registered_passengers.id = passengers_records.registered_passengers_id
+ WHERE age < 27 AND (point_of_embarkation= 'Q' OR point_of_embarkation = 'C');
+
+  
+/* Question 4: Query to get how many of the passengers that embarked at Southampton survived? 
+Passengers that embarked at Southampton survived = 218 */
+SELECT passengers_records.registered_passengers_id
+FROM passengers_records INNER JOIN accident_records 
+ON passengers_records.registered_passengers_id = accident_records.registered_passengers_id
+WHERE point_of_embarkation LIKE 'S' AND passenger_survived LIKE 1;
+
+
+/* Question 5: Query to get the id, name and the total number of passengers who paid a fare greater 
+than $100 and above the age of 35 had siblings or spouses on board.
+Total number of passengers = 9 */
```

Comment:
Write the query that came about you obtaining `9` as the result. Meanwhile, `9` is correct. But you didn't show me how you got 9